### PR TITLE
new slimmed version

### PR DESCRIPTION
**Pull request by erik78se**
* Remove all the manual steps.
* Remove unused package deps.
* Still missing ./go/pkg/mod/github.com/erigontech/silkworm-go@v0.14.0/lib/linux_x64/libsilkworm_capi.so

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -39,67 +39,16 @@ description: |
...
-      curl -L "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" -o /tmp/go${GO_VERSION}.linux-amd64.tar.gz
-      tar -C /usr/local -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz
-      export GOROOT=$GO_ROOT
-      export GOPATH=$GO_PATH
-      export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
-      $GO_BINARY version
   erigon:
-    after: [go-deps,wrappers]
     plugin: go
+    build-snaps: [go/latest/stable]
```
Comment (jakobilobi) [inline snap/snapcraft.yaml:44]: This implicitly does all of what was removed, kind of?

Comment (erik78se) [reply snap/snapcraft.yaml:44]: Yeah - mostly. But It needs to get back some of it since the Makefile is full of compiler flags that are needed etc.

**Comment by erik78se** [comment]
So I added back some of the original stuff that uses make since without the compiler flags, the built binaries won't work.

**Comment by erik78se** [comment]
This version also "only" ships the erigon binary, omitting the other binaries. This is likely not going to work, but the resulting snap becomes 600MB+ so I thought start small and see if this all works.

**Pull request merged**